### PR TITLE
fix(tooltip): create portal to add tooltip floating content in body context

### PR DESCRIPTION
**Pull request by ricardogoncalves89**

### CODE REVIEW

File: packages/ui-components/src/utils/types/components.ts [hunk 1]
```diff
@@ -14,6 +14,15 @@ export interface IAnchor {
 	download?: string;
 }
 
+export interface ICustomCss {
```
Comment (djcouto) [inline packages/ui-components/src/utils/types/components.ts:17]: 👌

**Review by djcouto — APPROVED** [review]
👏 🚀

**Review by pedrorebelocosta — APPROVED** [review]

File: packages/react-ui-components/src/stories/Components/Tooltip/Tooltip.stories.tsx [hunk 2]
```diff
@@ -28,7 +31,7 @@ export default {
 
 const TooltipTemplate: ComponentStory<typeof KvTooltip> = args => (
 	<KvTooltip {...args}>
-		<KvActionButton type={EActionButtonType.Primary}>Hover me!</KvActionButton>
+		<KvActionButtonText text={'Hover me!'} type={EActionButtonType.Primary} />
```
Comment (miguelpinto98) [inline packages/react-ui-components/src/stories/Components/Tooltip/Tooltip.stories.tsx:34]: text="Hover me!"

File: packages/ui-components/src/components/portal/portal.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+		return middleware;
+	};
+
+	private getOptions = (): Partial<ComputePositionConfig> => {
+		const middleware = this.getMiddlewareConfig();
+
+		return mergeWith({}, this.options, { middleware }, (objValue, srcValue) => {
+			if (isArray(objValue)) {
+				return unionWith(srcValue, objValue, (a, b) => a.name === b.name);
```
Comment (miguelpinto98) [inline packages/ui-components/src/components/portal/portal.tsx:74]: re-use this common code, please

**Review by miguelpinto98 — CHANGES_REQUESTED** [review]
well done! 👍 
I only left some comments to understand some points, after that we can continue.

**Review by miguelpinto98 — COMMENTED** [review]
since I will on vacation I will remove my requests

**Review by miguelpinto98 — APPROVED** [review]
👍

**Pull request merged**